test-data/unit/stubgen.test

```diff
@@ -595,6 +606,33 @@ class X(NamedTuple):
     a: Incomplete
     b: Incomplete
 
+[case testNamedtupleUsingInvalidIdent]
+import collections, x
+X = collections.namedtuple('X', ['@'])
```

Comment:
I don't think that it is correct. It just drops a field that is present in the original code.

I think a proper way to fix is to use `X = collections.namedtuple('X', ['@'])` in stubs.

There are several typeshed examples where we actually do this.

Reply:
Generating X = collections.namedtuple('X', ['@', 'a', 'b', 'c']) in the stubs would create invalid stub code.
Well imho I prefer partial correctness (and correct code), since we could resolve other fields (a, b, c in my case) and skip incorrect one.